dts/bindings/pwm/nxp,pca9685.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2022, Nick Ward <nix.ward@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP PCA9685 16-channel, 12-bit PWM Fm+ I2C-bus LED controller
+
+compatible: "nxp,pca9685"
+
+include: [pwm-controller.yaml, i2c-device.yaml, base.yaml]
+
+properties:
+  reg:
+    required: true
+
+  outdrv_open_drain:
```

Comment:
rename to `open-drain` to match linux binding.

Reply:
Done